invoke.py

```diff
@@ -36,7 +36,7 @@ def __extract_runtime_configuration(config):
     from colorama import init, Fore
     init(autoreset=True)
 
-    config['root_dir'] = os.path.dirname(os.path.abspath(__file__))
+    config['root_dir'] = os.path.dirname(os.path.abspath("__file__"))
```

Comment:
This looks weird. Are you sure?

Reply:
Sorry I failed my stash, I pushed an earlier non-working version

Reply:
Is the PR okay now?

Reply:
Yes it is, sorry again :sweat_smile: